test/test-sauce-labs.js

```diff
@@ -0,0 +1,222 @@
+/* jshint esversion: 6 */
```

Comment:
Yeah, I think that's a good idea. We'll have to extract out the platforms and maybe some other options into a json config probably.